search-app/src/cmr/search/validators/validation.clj

```diff
@@ -18,7 +19,7 @@
   "Returns the umm versioned result formats for the given concept-type"
   [concept-type]
   (for [format-key [:umm-json :umm-json-results]
-        version (umm-version/versions concept-type)]
+        version ((merge umm-version/versions generic/documents-all-versions) concept-type)]
```

Comment:
There is something weird going on here that is resulting in two `((` to start,  I think it would be better to use `(get`